.github/workflows/build-x86-image.yaml

```diff
@@ -528,8 +528,12 @@ jobs:
       - name: Load image
         run: docker load --input kube-ovn.tar
 
-      - name: Export debug image tag
-        run: echo "DEBUG_TAG='$(cat VERSION)-debug'" >> "$GITHUB_ENV"
+      - name: Set environment variables
+        run: |
+          if [ $(($RANDOM%2)) -ne 0 ]; then
```

Comment:
what does the RANDOM mean here?

Reply:
A random switch to enable valgrind memleak check.

Reply:
Valgrind memleack check cannot be enabled if ovs/ovn is running as a non-root user. But we want to test both running as non-root and memleak check.

Reply:
we need a comment to clarify this logical

Reply:
done